internal/xds/translator/translator.go

```diff
@@ -252,6 +260,16 @@ func (t *Translator) processHTTPListenerXdsTranslation(tCtx *types.ResourceVersi
 			return err
 		}
 
+		// Create oauth2 token endpoint clusters, if needed.
+		if err := createOAuth2TokenEndpointClusters(tCtx, httpListener.Routes); err != nil {
+			return err
+		}
+
+		// Create oauth2 client and HMAC secrets, if needed.
```

Comment:
curious, does this order matter ?

Reply:
I think the order here doesn't matter because DeltaXDS will eventually make the order correct when pushing all the resources to Envoy